flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/detail/job-checkpoints-detail.component.html

```diff
@@ -21,6 +21,8 @@
     <strong>Path:</strong> {{checkPointDetail?.external_path || '-'}}
     <nz-divider nzType="vertical"></nz-divider>
     <strong>Discarded:</strong> {{checkPointDetail?.discarded || '-'}}
+    <nz-divider nzType="vertical"></nz-divider>
+    <strong>Checkpoint Type:</strong> {{checkPointDetail?.checkpoint_type === "CHECKPOINT" ? (checkPointConfig?.unaligned_checkpoints ? 'unaligned checkpoint' : 'aligned checkpoint') : (checkPointDetail?.checkpoint_type === "SYNC_SAVEPOINT" ? 'savepoint on cancel' : 'savepoint')|| '-'}}
```

Comment:
Hi @gm7y8 
it would be better to move this calculation into typescript file as a getter

Reply:
makes sense. I just moved this into function in the ts file.